compatibility/bazel-haskell-deps.bzl

```diff
@@ -37,11 +37,16 @@ def daml_haskell_deps():
         local_snapshot = "//:stack-snapshot.yaml",
         packages = [
             "base",
+            "bazel-runfiles",
```

Comment:
Better use the version shipped with `rules_haskell`: `@rules_haskell//tools/runfiles`. We want the runfiles library in sync with `rules_haskell` rather than the Stackage snapshot.
```suggestion
```